proposals/KEP-0002-github-actions-workflow-alignment/README.md

```
@@ -0,0 +1,163 @@
+  - Add `cosign` signing and attestation
+
+So that we can ensure for all workflows:
+- Multi-arch builds: `linux/arm64`, `linux/amd64` with QEMU setup (extending it in the future as needed)
+- `docker/metadata-action`: Rich metadata generation
+- `docker/build-push-action`: Standardized build action
+- Image signing: Using `cosign sign` with _digest_ references
+- SBOM generation: Using `anchore/sbom-action` for SPDX format
+- SBOM attestation: Using `cosign attest` (instead of `cosign attach`)
+- Digest-based references: All signing/attestation uses `@sha256:...`
+- Permissions:
+   - Add `id-token: write` as needed by Cosig
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)